chapter_10/10-8.py

```diff
@@ -0,0 +1,11 @@
+from pathlib import Path
+
+files = [Path("cats.txt"), Path("dogs.txt")]
+
+for file in files:
+    try:
+        filedump = f"{file} names: \n"
+        filedump += file.read_text()
+        print(filedump)
+    except FileNotFoundError:
+        print(f"Error {file} not found")
```

Comment:
```suggestion
        print(f"Error {file} not found")
        
```
All it needs is a new line at the end of the file. Otherwise, everything works perfect!